Feature5

pinak22dhir opened the pull request:
added food ordering system,resolve issue #115

shrey141102 commented:
Please put it inside a folder, and share screenshots

shrey141102 commented:
Please put it inside a folder, and share screenshots

shrey141102 commented:
Its an empty file?

pinak22dhir commented:
it is a simple project and did not have images of food just a class code, i made the changes now it is visible to you?

shrey141102 commented:
> it is a simple project and did not have images of food just a class code, i made the changes now it is visible to you?

can u show input/output?

shrey141102 commented:
changes are visible 👍🏻

pinak22dhir commented:
yes @shrey141102 , it is too visible when you run the code

shrey141102 commented:
> yes @shrey141102 , it is too visible when you run the code

I cant run it without merging..
Can you please follow the PR template that i have put?
thank you

shrey141102 commented:
because every other PR is done in the same way

pinak22dhir commented:
Describe your changes :
added food_orderin.py
Issue ticket number and link:
resolves issue #115 
Screenshots of your project output, whatever you have added
![Screenshot 2024-01-09 234453](https://github.com/shrey141102/PythonProjects/assets/122675390/df399554-a801-4888-bbd0-605515572aa4)


Checklist before requesting a review
 I have performed a self-review of my code
 If it is a new project, I have done thorough tests.
 Have you updated the README file of the repository and added the name of the project in it?
 Have you added a README.md file inside the directory of your project and explained it there properly (with images)?
